guides/source/active_model_basics.md

```diff
@@ -3,31 +3,53 @@
 Active Model Basics
 ===================
 
-This guide should provide you with all you need to get started using model
-classes. Active Model allows for Action Pack helpers to interact with
-plain Ruby objects. Active Model also helps build custom ORMs for use
+This guide will provide you with what you need to get started using Active
+Model. Active Model provides a way for Action Pack and Action View helpers to
+interact with plain Ruby objects. It also helps to build custom ORMs for use
 outside of the Rails framework.
 
 After reading this guide, you will know:
 
-* How an Active Record model behaves.
-* How Callbacks and validations work.
-* How serializers work.
-* How Active Model integrates with the Rails internationalization (i18n) framework.
+* What Active Model is?
```

Comment:
I don't think you need the question mark at the end of this first point. 'What Active Model is' is a sentence in its own right.